<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>556</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="labelTitulo">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>0</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>17</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Listar Cadastro</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="campoSUS">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>90</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelCadastroSUS">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>90</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Cadastro SUS</string>
    </property>
   </widget>
   <widget class="QPushButton" name="botaoConsultar">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>150</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Consultar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="botaoListarTodos">
    <property name="geometry">
     <rect>
      <x>174</x>
      <y>150</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Listar Todos</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>556</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
